### PR TITLE
Action Bar and Video Workspace visual updates

### PR DESCRIPTION
**Pull request by eldiegod**
Fixes: #1673 and #1818

### CODE REVIEW

**Comment by kdembler** [comment]
@eldiegod This needs assigned reviewer

**Review by drillprop — CHANGES_REQUESTED** [review]
I think we need to add box-shadow on top of the tabs bar as well: 
Look at the designs: 
image
Yours: 
image

**Comment by eldiegod** [comment]
> I think we need to add box-shadow on top of the tabs bar as well: Look at the designs: image Yours: image

@drillprop  well spotted 👍

File: src/views/studio/VideoWorkspace/VideoWorkspaceTabsBar/VideoWorkspaceTabsBar.styles.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import styled from '@emotion/styled'
 
 import { Pill } from '@/components/Pill'
 import { Text } from '@/components/Text'
-import { media, oldColors, sizes } from '@/styles'
+import { cVar, media, oldColors, sizes } from '@/styles'
```
Comment (drillprop) [inline src/views/studio/VideoWorkspace/VideoWorkspaceTabsBar/VideoWorkspaceTabsBar.styles.ts]: Sorry to bother you again 😅  but could we also replace all usages of `oldColors` to `cVar` in this file?

**Review by drillprop — CHANGES_REQUESTED** [review]

File: src/views/studio/VideoWorkspace/VideoWorkspaceTabsBar/VideoWorkspaceTabsBar.styles.ts (hunk 1 above)
Comment (eldiegod) [reply src/views/studio/VideoWorkspace/VideoWorkspaceTabsBar/VideoWorkspaceTabsBar.styles.ts]: it has been done 👍

Comment (drillprop) [reply src/views/studio/VideoWorkspace/VideoWorkspaceTabsBar/VideoWorkspaceTabsBar.styles.ts]: You removed `oldColors` in `VideoWorkspace.styles` and I was pointing to `VideoWorkspaceTabsBar.styles` 😅

Comment (eldiegod) [reply src/views/studio/VideoWorkspace/VideoWorkspaceTabsBar/VideoWorkspaceTabsBar.styles.ts]: oopsy daisy, remove them as well now

**Review by drillprop — APPROVED** [review]

File: src/views/studio/VideoWorkspace/VideoWorkspace.style.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ export const DrawerOverlay = styled.div`
   right: 0;
   bottom: 0;
   left: 0;
-  background-color: ${oldColors.transparentBlack[54]};
+  background-color: ${cVar('colorCoreNeutral500Darken')};
```
Comment (kdembler) [inline src/views/studio/VideoWorkspace/VideoWorkspace.style.ts]: Shouldn't this use a color token instead of `colorCore`?

Comment (eldiegod) [reply src/views/studio/VideoWorkspace/VideoWorkspace.style.ts]: 100%

**Pull request merged**